vector/src/main/java/im/vector/app/features/call/transfer/CallTransferActivity.kt

```diff
@@ -82,25 +79,32 @@ class CallTransferActivity : VectorBaseActivity<ActivityCallTransferBinding>() {
             when (views.callTransferTabLayout.selectedTabPosition) {
                 CallTransferPagerAdapter.USER_LIST_INDEX -> {
                     val selectedUser = sectionsPagerAdapter.userListFragment?.getCurrentState()?.getSelectedMatrixId()?.firstOrNull() ?: return@debouncedClicks
-                    val action = CallTransferAction.ConnectWithUserId(views.callTransferConsultCheckBox.isChecked, selectedUser)
-                    callTransferViewModel.handle(action)
+                    val result = CallTransferResult.ConnectWithUserId(views.callTransferConsultCheckBox.isChecked, selectedUser)
+                    handleComplete(result)
```

Comment:
to double check my understanding, is this this line the fix? (by returning the action as a result to the starting activity so that we can handle with the original `VectorCallViewModel`)

Reply:
yup!

Reply:
Well actually there was 2 parts to the fix, this fixed the second part. I reformatted the description of the PR to make ti clearer what the 2 parts were.